ufl/finiteelement/elementlist.py

```diff
@@ -150,6 +150,7 @@ def show_elements():
                  "double contravariant Piola", (2, 2), ("triangle", "tetrahedron"))
 register_element("Conforming Arnold-Winther", "AWc", 2, HDivDiv,
                  "double contravariant Piola", (3, None), ("triangle", "tetrahedron"))
+register_element("C0 Modified", "C0Mod", 0, H1, "identity", (3, 3), simplices[0])
```

Comment:
Prefer not to do this at all, the "variant" mechanism can allow us to specify the difference.